### PR TITLE
Initial commit of inventory-manager.lic. 

### PR DESCRIPTION
**Pull request by jayRyan24**
This script is intended to be used as an alt inventory management tool. You can add all your characters `inv full` to it. It will then let you print out their inventory from any character. You can also search all characters for specific keywords, etc. 

Eventually wanting to add some vault support, but that's a bigger task.

### CODE REVIEW

**Comment by jayRyan24** [comment]
image

image

image

image

**Comment by jayRyan24** [comment]
Updated them to show whether the item is worn or in container based on suggestion from user.

image

File: inventory-manager.lic [hunk 1]
```diff
@@ -0,0 +1,125 @@
+=begin
+  Documentation: https://elanthipedia.play.net/Lich_script_repository#invdb
```
Comment (rpherbig) [inline inventory-manager.lic:2]: 💡 `inventory-manager`?

File: inventory-manager.lic [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+			],
+			[
+				{ name: 'remove', regex: /remove/i, description: "Remove character's inventory." },
+				{ name: 'name', regex: /\w+/i, optional: false, description: 'Character to remove.' }
+      ]
+    ]
+
+    args = parse_args(arg_definitions)
+
+    pause
```
Comment (rpherbig) [inline inventory-manager.lic:32]: 💡 Why `pause` here?

File: inventory-manager.lic [hunk 3]
```diff
@@ -0,0 +1,125 @@
...
+	  if args.save
+		  add_current_inv
+	  elsif args.remove
+      remove_character_data(args.name)
+	  elsif args.search
+      search_for_item(args.item)
+    elsif args.list
+      list_character_inv(args.name)
+    else
+			message 'Type ;invdb help for a usage guide'
```
Comment (rpherbig) [inline inventory-manager.lic:45]: 💡 `... ;inventory-manager ...`?

File: inventory-manager.lic [hunk 4]
```diff
@@ -0,0 +1,125 @@
+=begin
+  Documentation: https://elanthipedia.play.net/Lich_script_repository#invdb
+=end
+
+class InventoryManager
+  include DRC
```
Comment (rpherbig) [inline inventory-manager.lic:6]: 💡 Should this `custom_require` at least `common`? Maybe also `drinfomon` if that's where `Char.name` comes from.

File: inventory-manager.lic [hunk 5]
```diff
@@ -0,0 +1,125 @@
...
+        item.concat(' (inside container)')
+      end
+      @item_data[@active_character] << item
+    end
+
+    message "Saving data for #{@active_character}!"
+    File.open(@item_db_path, 'w') { |file| file.write(@item_data.to_yaml) }
+  end
+
+  def list_character_inv(name = '')
```
Comment (rpherbig) [inline inventory-manager.lic:87]: 💡 I'm not a fan of a default arg here, it should always be provided

File: inventory-manager.lic (hunk 4 above)
Comment (jayRyan24) [reply inventory-manager.lic:6]: Yes, I am not sure how I forgot that.

File: inventory-manager.lic (hunk 3 above)
Comment (jayRyan24) [reply inventory-manager.lic:45]: Yeah, I renamed. I thought ;inventory-manager was a script on github so I renamed it. Then realized I had done this before... and just removed the old copy.

File: inventory-manager.lic (hunk 2 above)
Comment (jayRyan24) [reply inventory-manager.lic:32]: I am not sure, I've seen other scripts do this within dependency so I added it. Figured it was some type of "thing".

File: inventory-manager.lic (hunk 1 above)
Comment (jayRyan24) [reply inventory-manager.lic:2]: Leftover, will edit.

File: inventory-manager.lic (hunk 5 above)
Comment (jayRyan24) [reply inventory-manager.lic:87]: That's a leftover from a previous iteration. I will make that change.

**Pull request merged**